apps/framework-cli/src/cli/routines/metrics_console/run_console/client.rs

```diff
@@ -76,6 +91,32 @@ pub async fn getting_metrics_data() -> Result<ParsedMetricsData> {
                 request_count: count_value,
                 path: (metrics_vec[j].labels["path"]).to_string(),
             });
+        } else if metrics_vec[j].metric == "bytes_in_total" {
+            let value: f64 = match &metrics_vec[j].value {
+                prometheus_parse::Value::Counter(v) => *v,
+                prometheus_parse::Value::Untyped(v) => *v,
+                _ => 0.0,
+            };
+
+            total_bytes_in += value as u64;
+
+            paths_bytes_in_vec.push(PathBytesData {
+                path: (metrics_vec[j].labels["path"]).to_string(),
+                bytes_data: value as u64,
+            });
+        } else if metrics_vec[j].metric == "bytes_out_total" {
+            let value: f64 = match &metrics_vec[j].value {
+                prometheus_parse::Value::Counter(v) => *v,
+                prometheus_parse::Value::Untyped(v) => *v,
+                _ => 0.0,
+            };
+
+            total_bytes_out += value as u64;
+
+            paths_bytes_out_vec.push(PathBytesData {
+                path: (metrics_vec[j].labels["path"]).to_string(),
+                bytes_data: value as u64,
```

Comment:
In the 2 branches the logic seems to be the same but applied to a different vector?